pyre/zbeacon.py

```diff
@@ -36,28 +35,28 @@
 logger = logging.getLogger(__name__)
 
 INTERVAL_DFLT = 1.0
-BEACON_MAX = 255      # Max size of beacon data
+BEACON_MAX = 255  # Max size of beacon data
 MULTICAST_GRP = '225.25.25.25'
-ENETDOWN = 50   #socket error, network is down
-ENETUNREACH = 51 #socket error, network unreachable
+ENETDOWN = 50  # socket error, network is down
+ENETUNREACH = 51  # socket error, network unreachable
 
 
 class ZBeacon(object):
 
     def __init__(self, ctx, pipe, *args, **kwargs):
-        self.ctx = ctx                #  ZMQ context
-        self.pipe = pipe              #  Actor command pipe
+        self.ctx = ctx  # ZMQ context
```

Comment:
If you can please undo the whitespace changes? It has nothing to do with the change and it renders the change history useless in case of tracing changes.